### PR TITLE
Refactor KTR and KTRLite Tests

### LINKED ISSUE #472: `obs_sigma` in KTRLite model
`obs_sigma` in KTRLite model seems not to be stable. It depends on the training data and the `span_level`. Is there a way to stabilize this parameter?

### PR DESCRIPTION
**Pull request by edwinnglabs**
## Description

Refactor unit test for KTR and KTRLite

### CODE REVIEW

File: orbit/template/ktrlite.py [hunk 1]
```diff
@@ -194,8 +194,11 @@ def _set_default_args(self):
         elif not isinstance(self._seasonality, list) and isinstance(self._seasonality, (int, float)):
             self._seasonality = [self.seasonality]
 
+        # set some defaults for seasonality_fs_order
         if self._seasonality and self._seasonality_fs_order is None:
-            self._seasonality_fs_order = [2] * len(self._seasonality)
+            self._seasonality_fs_order = list()
+            for s in self._seasonality:
+                self._seasonality_fs_order.append(math.floor(s / 2))
```
Comment (wangzhishi) [inline orbit/template/ktrlite.py:201]: i dont see why this is a better default. e.g., seasonality=12, order would be 6. and there will be 12 fourier regressors. 1. is it too much? 2. at most it should be 11?

Comment (edwinnglabs) [reply orbit/template/ktrlite.py:201]: i keep forgetting the 2 times factor. I actually mean / 4 in that case.

**Review by wangzhishi — APPROVED** [review]
LGTM

**Review by wangzhishi — APPROVED** [review]
LGTM

**Pull request merged**